frontend/interfaces/policy.ts

```diff
@@ -36,7 +36,7 @@ export interface IPolicy {
   author_email: string;
   resolution: string;
   platform: SelectedPlatformString;
-  team_id?: number;
+  team_id?: number | null;
```

Comment:
Nice catch, team_id will always be a number or null now, instead of undefined, like in the case of listing global policies from `/global/policies`, updating this